src/main/java/com/puppycrawl/tools/checkstyle/api/AbstractLoader.java

```diff
@@ -49,12 +49,6 @@
  */
 public abstract class AbstractLoader
     extends DefaultHandler {
-    /** Feature that enables loading external DTD when loading XML files. */
-    private static final String LOAD_EXTERNAL_DTD =
-        "http://apache.org/xml/features/nonvalidating/load-external-dtd";
```

Comment:
@Nimfadora This is not true. Look at blame for these lines and associated issue.
https://github.com/checkstyle/checkstyle/blame/154f120d7b71c30ec64f3ef801fb080a24a21c4a/src/main/java/com/puppycrawl/tools/checkstyle/api/AbstractLoader.java#L52
https://github.com/checkstyle/checkstyle/issues/3605

This removal must be reversed.